qemu/qmp.go

```diff
@@ -912,15 +912,21 @@ func (q *QMP) ExecuteNetPCIDeviceAdd(ctx context.Context, netdevID, devID, macAd
 	args := map[string]interface{}{
 		"id":      devID,
 		"driver":  VirtioNetPCI,
-		"netdev":  netdevID,
-		"mac":     macAddr,
-		"addr":    addr,
 		"romfile": romfile,
```

Comment:
@sboeuf unrelated. I wish we could do something like this for the rom file too. If unspecified do not set it. If specified set it to what was specified. However here != "" will be wrong comparison as we need to support setting it to "". That way we are breaking any existing implementations before we started supporting the rom file.

Reply:
@mcastelino yeah me too... That's because they decided to specify `efi-virtio.rom` by default...